src/coefficients/stdfunction_coefficient.hpp

```diff
@@ -60,6 +60,19 @@ class StdFunctionVectorCoefficient : public mfem::VectorCoefficient {
   std::function<void(mfem::Vector &, mfem::Vector &)> m_func;
 };
 
+/**
+   \brief MakeTrueEssList takes in a FESpace, a vector coefficient, and produces a list
+   of essential boundary conditions
+
+   \param[in] pfes A finite element space for the constrained grid function
+   \param[in] c A VectorCoefficient that is porjected on to the mesh. All
```

Comment:
... projected...

Reply:
I guess my spelling mistakes finally got caught the second time around. Typing is hard. I'll fix these.